api/app/config/detekt.yml

```diff
@@ -49,6 +49,8 @@ style:
     active: false
   UnusedPrivateMember:
     active: false
+  ThrowsCount:
+    active: false
```

Comment:
Detekt was complaining about too many throws statement, and honestly I find this style rule questionable. Let me know if you would prefer me to abide by it and refactor the code to comply instead.

Reply:
Agreed, I think we probably just haven't triggered this one before..